.CI/Test/Common.c

```diff
@@ -1,4 +1,4 @@
-#include "ModelicaUtilities.h"
+#include "../../Modelica/Resources/C-Sources/ModelicaUtilities.h"
```

Comment:
(Attaching comment to one of several possible places.)

Considering that _ModelicaUtilities.h_ has prototypes for functions that should be provided by the tool, and not for functions provided by the MSL, wouldn't a mode natural location be inside ModelicaServices?
```suggestion
#include "../../ModelicaServices/Resources/Include/ModelicaUtilities.h"
```

(Also note the use of the default include directory for the library.)